src/main/java/com/xceptance/xlt/engine/httprequest/HttpRequest.java

```diff
@@ -390,7 +409,7 @@ public HttpRequest param(final String name, String value)
      */
     public HttpRequest params(final List<NameValuePair> params)
     {
-        params.forEach(p -> param(p.getName(), p.getValue()));
+        params.forEach(p -> param(p));
```

Comment:
Might be shortened by using method reference: `params.forEach(this::param);`
WDYT?

Reply:
Done.